public/styles/articlepage.scss

```diff
@@ -0,0 +1,256 @@
+.site-content {
+  margin-bottom: 100px;
```

Comment:
Can you convert the 100px to em? Px is an absolute unit of measurement while em is a relative unit of measurement. em suits many screens better than px. 16 px is equal to 1em.